### ISSUE TITLE
Cannot build Emscripten samples in Linux (Debian)

### ISSUE THREAD

**Issue by AndrewAMD** [floooh/sokol-samples#23]
Hi,

I am having trouble building sokol-samples for Emscripten.  I have produced some logs, I am not sure what the problem is.

```
> python --version
Python 2.7.13

> cmake --version
cmake version 3.7.2

> make --version
GNU Make 4.1

```

The procedure:

```
mkdir ~/fips-workspace/
cd ~/fips-workspace/

git clone https://github.com/floooh/sokol
git clone https://github.com/floooh/sokol-samples

cd sokol-samples
./fips setup emscripten > ./emout.txt 2>./emerr.txt
./fips set config webgl2-emsc-make-release
./fips build > ./bout.txt 2>./berr.txt

```

Here are the logs:
[berr.txt](https://github.com/floooh/sokol-samples/files/2932236/berr.txt)
[bout.txt](https://github.com/floooh/sokol-samples/files/2932237/bout.txt)
[emerr.txt](https://github.com/floooh/sokol-samples/files/2932238/emerr.txt)
[emout.txt](https://github.com/floooh/sokol-samples/files/2932239/emout.txt)

Thanks,
Andrew

**Comment by floooh** [comment]
Yeah, whoops, I can reproduce this with the webgl2-emsc-make-release build config, looks like I haven't tested this one for a while.

I'll try to look into it tomorrow, it's a bit late now here. In the meantime, you can try to build the sokol_app.h samples via emscripten, these should work (these are the samples used on the samples webpage):

```
> ./fips set config sapp-webgl2-wasm-ninja-release
> ./fips build
> ./fips run triangle-sapp
```

(you'll need the ninja build tool for that, since I haven't created make-based build configs for the sokol-samples, I'm not sure what the Debian package is called, IFIR it was something like "ninja-buildtool").
